server/controllers/accesses.js

```diff
@@ -76,24 +76,32 @@ export async function create(req, res) {
 // Fetch operations using the backend. Note: client needs to get the operations
 // back.
 export async function fetchOperations(req, res) {
+    let access = req.preloaded.access;
     try {
         // Fetch operations
-        let access = req.preloaded.access;
         await commonAccountManager.retrieveOperationsByAccess(access);
         res.sendStatus(200);
     } catch (err) {
+        if (err.errCode) {
+            access.lastImportStatus = err.errCode;
+            await access.save();
```

Comment:
What if `access.save` fails? There won't be anything that saves it!
Also, to be more effectful, I'd have the lastImportStatus be saved directly in the account manager, so that this also works for the initial recurrent polling. Plus, this is going to avoid some code duplication :)